### ISSUE TITLE
Unable to use Google Auth

### ISSUE THREAD

**Issue by mast3rd3mon** [EddyVerbruggen/nativescript-plugin-firebase#1573]
I'm trying to enable Google Auth within my app, but whenever I try to login I get an error saying that I need to upload my SHA1 fingerprint. I have uploaded my SHA1 fingerprint from both the App Signing tab in the Google Play Console, and my SHA1 fingerprint from local debug.keystore file.

Both of these keys appear in the google-services.json file, so my guess is that the plugin can't find them? I haven't tried it on iOS, only on Android.

Info:
nativescript: 6.3.3
tns-core-modules: 6.3.2
tns-android: 6.3.1
plugin version: 10.5.0

**Comment by manojdcoder** [comment]
Technically speaking the plugin do not have to do anything about fingerprints. It's all handled by the native SDK internally. But the plugin may throw similar error message only when it receives correspondent error code in the callbacks.

Did you try a clean build to ensure your updated google-service.json file made it to build?
Can you confirm the command you are using to build the app?

**Comment by mast3rd3mon** [comment]
Yeah, its copied into my platforms folder. And the command I'm using is `tns run android`

**Comment by manojdcoder** [comment]
By `debug.keystore`, are you referring to Androiod's default debug keystore? If Yes, did you double check the SHA1 - whether the one APK built with matches with one from the keystore file you are referring to?

**Comment by saleemepoch** [comment]
@mast3rd3mon in your login function remove the entire googleOptions section and then try to login.

**Comment by mast3rd3mon** [comment]
@manojdcoder I don't know how to do that?

@saleemepoch I don't have the googleOptions section in my login function, only ```type: firebase.LoginType.GOOGLE```

**Comment by manojdcoder** [comment]
@mast3rd3mon You could google that, [here](https://stackoverflow.com/questions/11331469/how-do-i-find-out-which-keystore-was-used-to-sign-an-app) is a good SO post.
